tests/fixtures/requests/presentation_with_filename.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<modules>
+    <module name="presentation">
+        <document url="https://test-install.blindsidenetworks.com/default.pdf" filename="presentation.pdf"/>
```

Comment:
This file is no more available.

Reply:
yes, the URLs should be valid.
But I am sure I covered all / most of them in other PRs. 
Can we merge all the open PRs and re-check the URLs then? I created a reminder (see issue #255)